vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -98,6 +101,24 @@
 namespace react {
 namespace uwp {
 
+namespace {
+
+bool HasPackageIdentity() noexcept {
+  static const bool hasPackageIdentity = []() noexcept {
+    auto packageStatics = winrt::get_activation_factory<winrt::Windows::ApplicationModel::IPackageStatics>(
+        winrt::name_of<winrt::Windows::ApplicationModel::Package>());
+    auto abiPackageStatics =
+        static_cast<ABI::Windows::ApplicationModel::IPackageStatics *>(winrt::get_abi(packageStatics));
+    winrt::com_ptr<ABI::Windows::ApplicationModel::IPackage> dummy;
+    return abiPackageStatics->get_Current(reinterpret_cast<ABI::Windows::ApplicationModel::IPackage **>(
+               winrt::put_abi(dummy))) != APPMODEL_ERROR_NO_PACKAGE;
+  }();
```

Comment:
I think this function will eventually be used in many other places for the same reason. Can you put it in a separate helper file.

Reply:
YAGNI 🙂. We'll move it when/if we need to.